api/axis.py

```diff
@@ -443,13 +456,17 @@ def process_one(self, e, namespaces):
             expiration_date = expiration_date.text
             expiration_date = datetime.strptime(
                 expiration_date, self.FULL_DATE_FORMAT)
-            
+
+        # WTH??? Why is identifier None? Is this ever used?
         fulfillment = FulfillmentInfo(
+            collection=None, data_source_name=DataSource.AXIS_360,
```

Comment:
I know I originally wrote this code but it might be better to just change these ResponseParser subclasses to take a collection in their constructor. In the tests we can pass in `collection=None` and it should work. This would let us get rid of the code that fills in `collection` later.